README.md

```diff
@@ -130,22 +135,24 @@ Sign your HTTP request with a defined client credential:
 ```java
 HttpTransport httpTransport = new ApacheHttpTransport();
 HttpRequestFactory requestFactory = httpTransport.createRequestFactory();
-URI uri = URI.create("https://endpoint.net/billing-usage/v1/reportSources");
+URI uri = URI.create("https://akaa-baseurl-xxxxxxxxxxx-xxxxxxxxxxxxx.luna.akamaiapis.net/billing-usage/v1/reportSources");
```

Comment:
I'm going to leave the real hostname in the example or avoid confusion. This was brought up by one of the test users. We don't currently explain that we grab the real hostname out of the `ClientCredential` and ignore this one. We can change this later if we come up with a better way to document that feature.